sdl/linux.c

```diff
@@ -18,7 +18,7 @@ typedef char     TEXT;
 int32 gui_req( uint32 img, TEXT *title, TEXT *reqtxt, TEXT *buttons )
 {
   GtkWidget *dialog = NULL;
-  GtkButtonsType btns = strcmp((char*)btns, "OK") ? GTK_BUTTONS_OK_CANCEL : GTK_BUTTONS_OK;
+  GtkButtonsType btns = g_strcmp0((char*)btns, "OK") ? GTK_BUTTONS_OK_CANCEL : GTK_BUTTONS_OK;

```

Comment:
I don't think this is the correct fix.

It should be doing a normal strcmp, but on "buttons" rather than the uninitialised "btns".